packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxis.ts

```diff
@@ -155,13 +158,15 @@ export const useChartCartesianAxis: ChartPlugin<UseChartCartesianAxisSignature<a
       return () => {};
     }
 
-    const handleMouseClick = (event: MouseEvent) => {
-      event.preventDefault();
+    const axisClickHandler = instance.addInteractionListener('drag', (state) => {
```

Comment:
Same feeling about forcing the usage of use-gesture events for stuff they are not planned for

Reply:
Drag is meant to detect `tap/click` as well.